src/main/java/edu/harvard/iq/dataverse/Dataverse.java

```diff
@@ -590,8 +590,17 @@ public void setCitationDatasetFieldTypes(List<DatasetFieldType> citationDatasetF
         this.citationDatasetFieldTypes = citationDatasetFieldTypes;
     }
     
-    
+    @Column(nullable = true)
+    private Boolean filePIDsEnabled;
 
+    public Boolean getFilePIDsEnabled() {
+        return filePIDsEnabled;
```

Comment:
Shouldn't this code return a default if null? If this should be optional (so this is handled as business logic in service layer), maybe return `Optional.ofNullable()`?

Reply:
I specifically wanted it to be nullable, and to be null by default; and I wanted the method above to return the actual value (or lack thereof) in the database table. Null value in my implementation has a meaning, of "not specifically configured" one way or another, indicating that the setting of the ancestral collection needs to be consulted instead (or the system-wide default, if none defined on the root collection). 
So yes, that was a long way of saying "business logic is elsewhere". I will add a javadoc entry.